File: Practica Teoria/MyApplicationCheckboxClase05Teoria/app/src/main/java/com/example/myapplicationcheckboxclase05teoria/MainActivity.java
package com.example.myapplicationcheckboxclase05teoria;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private CheckBox checkBoxLectura;
    private CheckBox checkBoxPasear;
    private CheckBox checkBoxSeriesTV;
    private CheckBox checkBoxCompras;

    private Button botonEnviar;
    private TextView textViewResultado;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        checkBoxLectura = findViewById(R.id.idLectura);
        checkBoxPasear = findViewById(R.id.idPasear);
        checkBoxSeriesTV = findViewById(R.id.idSeriesTv);
        checkBoxCompras = findViewById(R.id.idCompras);
        textViewResultado = findViewById(R.id.textViewResultdo);
        botonEnviar = findViewById(R.id.buttonEnviar);

        botonEnviar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                StringBuilder stringBuilder01 = new StringBuilder();

                stringBuilder01.append(checkBoxLectura.getText().toString() + ": " + checkBoxLectura.isChecked() + "\n");
                stringBuilder01.append(checkBoxPasear.getText().toString() + ": " + checkBoxPasear.isChecked() + "\n");
                stringBuilder01.append(checkBoxSeriesTV.getText().toString() + ": " + checkBoxSeriesTV.isChecked() + "\n");
                stringBuilder01.append(checkBoxCompras.getText().toString() + ": " + checkBoxCompras.isChecked() + "\n");

                textViewResultado.setText(stringBuilder01);
            }
        });
    }
}

File: Practica Teoria/MyApplicationPractica06DSM104/app/src/main/java/com/example/myapplicationpractica06_dsm104/MainActivity.kt
package com.example.myapplicationpractica06_dsm104

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RelativeLayout

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClickFrame(v: View?)
    {
        val llamar = Intent(this, FrameLayout::class.java)
        startActivity(llamar)
    }

    fun onCLickLinear(v: View?)
    {
        val llamar = Intent(this, LinearLayout::class.java)
        startActivity(llamar)
    }

    fun onCLickRelative(v: View?)
    {
        val llamar = Intent(this, RelativeLayout::class.java)
        startActivity(llamar)
    }

    fun onCLickTable(v: View?)
    {
        val llamar = Intent(this, TableLayout::class.java)
        startActivity(llamar)
    }

    fun onCLickGrid(v: View?)
    {
        val llamar = Intent(this, GridLayout::class.java)
        startActivity(llamar)
    }
}